Zend/zend_ast.h

```diff
@@ -193,7 +193,7 @@ ZEND_API zend_ast *zend_ast_create(zend_ast_kind kind, ...);
 
 ZEND_API zend_ast *zend_ast_create_decl(
 	zend_ast_kind kind, uint32_t flags, uint32_t start_lineno, zend_string *doc_comment,
-	zend_string *name, zend_ast *child0, zend_ast *child1, zend_ast *child2
+	zend_string *name, zend_ast *child0, zend_ast *child1, zend_ast *child2, zend_ast *child3
```

Comment:
Instead of changing the API/ABI here, wouldn't it make more sense to use a ** for the children? Right now, this ABI can be changed for PHP 7, but not generally in any other case than a major release.